tests/snuba/search/test_backend.py

```diff
@@ -186,6 +186,17 @@ def test_query_with_environment(self):
             query='bar')
         assert set(results) == set([self.group2])
 
+    def test_multi_environments(self):
+        self.set_up_multi_project()
+        results = self.backend.query(
+            [self.project, self.project2],
+            environments=[
+                self.environments['production'],
+                self.environments['staging']
+            ])
+        print set(results)
```

Comment:
Style: Remove

Reply:
oops wonder why linter didn't catch this

thanks!